docs/weaviate/manage-collections/collection-aliases.mdx

```diff
@@ -0,0 +1,330 @@
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+:::note
+
+- An alias name must be unique and cannot match any existing collection or alias name
+- Multiple aliases can point to the same collection
+- Aliases can be used anywhere a collection name is expected
+  :::
+
+## List all aliases
+
+Retrieve all aliases in your Weaviate instance.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START ListAllAliases"
+      endMarker="# END ListAllAliases"
+      language="py"
+    />
+  </TabItem>
+
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+## List aliases for a specific collection
+
+Get all aliases that point to a specific collection.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START ListCollectionAliases"
+      endMarker="# END ListCollectionAliases"
+      language="py"
+    />
+  </TabItem>
+
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+## Get a specific alias
+
+Retrieve information about a specific alias.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START GetAlias"
+      endMarker="# END GetAlias"
+      language="py"
+    />
+  </TabItem>
+
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+## Update an alias
+
+Change the target collection that an alias points to. This operation is atomic and provides instant switching between collections.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START UpdateAlias"
+      endMarker="# END UpdateAlias"
+      language="py"
+    />
+  </TabItem>
+
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+:::tip Use case: Zero-downtime migration
```

Comment:
This is great. Some version of this could go into concepts maybe? Then it can be linked from different places.

Reply:
I will add it to the starter guide with the code example on how to achieve it